components/cluster/command/clean.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package command
+
+import (
+	operator "github.com/pingcap/tiup/pkg/cluster/operation"
+	"github.com/spf13/cobra"
+)
+
+func newCleanCmd() *cobra.Command {
+	cleanOpt := operator.Options{}
+	cleanALl := false
+
+	cmd := &cobra.Command{
+		Use: "clean <cluster-name>",
+		Long: `Cleanup a specified cluster without destroying it.
+	You can retain some nodes and roles data when cleanup the cluster, eg:
```

Comment:
Please use 4 spaces to indent this content instead of a <TAB>.
Another issue is better to demonstrate `--ignore-xxx` in an individual command and doesn't hybrid them.